services/web/server/src/simcore_service_webserver/director_v2/_handlers.py

```diff
@@ -130,9 +131,13 @@ async def start_computation(request: web.Request) -> web.Response:
             project_wallet_id = project_wallet.wallet_id
 
         # Check whether user has access to the wallet
-        wallet = await wallets_api.get_wallet_by_user(
+        wallet = await wallets_api.get_wallet_with_available_credits_by_user_and_wallet(
             request.app, req_ctx.user_id, project_wallet_id, req_ctx.product_name
         )
+        if wallet.available_credits <= ZERO_CREDITS:
```

Comment:
I wonder whether zero is the right number. IMO here we should have the minimal unit that allows to run something. 
Let me argue this with an example. Think of a shop of candies. The smallest candy costs 3 dollars (it is a candy store in Zurich Banhhofstr). If your wallet has 2 dollars, you should not be able to buy anything. 
My point here is that not-enough-credits should be with that of the item with minimum price. In our case might be something as the credits to run a simulation say for 1 min??  Which is indeed not zero!

Reply:
pls keep it in mind.